### PR TITLE
Arachne 1995/multi atlas

### PR DESCRIPTION
**Pull request by pavgra**

### CODE REVIEW

File: src/modules/ExternalResourceManager/components/AtlasList/ModalCreateEdit/container.jsx [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+  mergeProps(stateProps, dispatchProps, ownProps) {
+    return {
+      ...stateProps,
+      ...dispatchProps,
+      ...ownProps,
+      resetForm() {
+        dispatchProps.resetForm();
+        dispatchProps.clearAtlasData();
+      },
+      doSubmit(data) {
```
Comment (johnSamilin) [inline src/modules/ExternalResourceManager/components/AtlasList/ModalCreateEdit/container.jsx]: why not async/await?

File: src/modules/ExternalResourceManager/components/AtlasList/presenter.jsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+ * Product Owner/Architecture: Gregory Klebanov
+ * Authors: Pavel Grafkin
+ * Created: March 14, 2018
+ *
+ */
+
+import React from 'react';
+import BEMHelper from 'services/BemHelper';
+import Grid from 'components/Grid';
+import PageWrapper from 'modules/Admin/components/PageWrapper';
```
Comment (johnSamilin) [inline src/modules/ExternalResourceManager/components/AtlasList/presenter.jsx]: this should probably become a common component

File: src/modules/ExternalResourceManager/const.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  atlases: () => '/external-resource-manager/atlases',
+};
+
+const apiPaths = {
+  atlases: ({ id } = {}) => `/api/v1/atlases${id ? `/${id}` : ''}`,
+  atlasConnection: ({ id }) => `/api/v1/admin/atlases/${id}/connection`,
+};
+
+const imgs = {
+  sidebarIco: '/img/icons/Universal_Desktop/Navigation/Arachne_Desktop_icon-Data_Catalog.png',
```
Comment (johnSamilin) [inline src/modules/ExternalResourceManager/const.js]: Data catalog?

File: src/services/PageableUtils.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import { get } from './Utils';
+
+export default class PageableUtils {
+
+  static getSorting(state, defaultSortField = 'name', defaultSortOrder = PageableUtils.SORT_ASC) {
+    const location = state.routing.locationBeforeTransitions;
+    const sort = get(location, 'query.sort', `${defaultSortField},${defaultSortOrder}`).split(',');
```
Comment (johnSamilin) [inline src/services/PageableUtils.js]: shouldn't here be type also as the 4th param?

File: src/modules/ExternalResourceManager/components/AtlasList/Actions/container.jsx [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+class ActionsBuilder extends ContainerBuilder {
+
+  getComponent() {
+    return Actions;
+  }
+
+  getMapDispatchToProps() {
+    return {
+      openNewAtlasModal: () => ModalUtils.actions.toggle(modal.atlasDetails, true),
+      refresh: actions.router.reload,
```
Comment (johnSamilin) [inline src/modules/ExternalResourceManager/components/AtlasList/Actions/container.jsx]: why not **loadAtlasList: actions.externalResourceManager.atlases.query**?

File: src/modules/ExternalResourceManager/const.js (hunk 3 above)
Comment (pavgra) [reply src/modules/ExternalResourceManager/const.js]: sadly said, I do not have any others yet

**Pull request merged**